### ISSUE TITLE
Standalone cluster with X509 certs setup failed:System.InvalidOperationException: There is an error in XML document

### ISSUE THREAD

**Issue by yderi** [microsoft/service-fabric#793]
Hello every one
Trying to setup Standalone Service Fabric Cluster with X509 certificates.
Here is the command `.\CreateServiceFabricCluster.ps1 -ClusterConfigFilePath .\X509-3.json -AcceptEULA -Verbose` 
Getting an error: `...There is an error in XML document....`
Here is the some part of messages in powershell window
`PS C:\Fabric> .\CreateServiceFabricCluster.ps1 -ClusterConfigFilePath .\X509-3.json -AcceptEULA -Verbose
Trace folder already exists. Traces will be written to existing trace folder: C:\Fabric\DeploymentTraces
Creating Service Fabric Cluster...
If it's taking too long, please check in Task Manager details and see if Fabric.exe for each node is running. If not, please look at: 1. traces in DeploymentTraces directory and 2. traces in FabricLogRoot configured in ClusterConfig.json.
Trace folder already exists. Traces will be written to existing trace folder: C:\Fabric\DeploymentTraces
Running Best Practices Analyzer...
Best Practices Analyzer completed successfully.
Creating Service Fabric Cluster...
Processing and validating cluster config.
Configuring nodes.
Default installation directory chosen based on system drive of machine 'HPSFSEC2'.
Copying installer to all machines.
Configuring machine 'HPSFSEC2'.
Configuring machine 'HPSFSEC1'.
Configuring machine 'HPSFSEC0'.
System.InvalidOperationException: There is an error in XML document (139, 8). ---> System.Xml.Schema.XmlSchemaValidationException: The required attribute 'Value' is missing.
   at System.Xml.Schema.XmlSchemaValidator.SendValidationEvent(XmlSchemaValidationException e, XmlSeverityType severity)
   at System.Xml.Schema.XmlSchemaValidator.SendValidationEvent(String code, String arg)
   at System.Xml.Schema.XmlSchemaValidator.CheckRequiredAttributes(SchemaElementDecl currentElementDecl)
   at System.Xml.Schema.XmlSchemaValidator.ValidateEndOfAttributes(XmlSchemaInfo schemaInfo)
   at System.Xml.XsdValidatingReader.ProcessElementEvent()
   at System.Xml.XsdValidatingReader.Read()
   at System.Xml.XmlReader.MoveToContent()
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReader1.Read35_SettingsOverridesTypeSection(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReader1.Read37_ClusterManifestType(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReader1.Read278_ClusterManifest()
   --- End of inner exception stack trace ---
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)`
Config attached:
[X509-3.json.txt](https://github.com/Azure/service-fabric-issues/files/1907482/X509-3.json.txt)
And there is a long log reddish sheet.
[Fabric-X509_setup_err.txt](https://github.com/Azure/service-fabric-issues/files/1907573/Fabric-X509_setup_err.txt)

What I'm doing wrong?

**Comment by maburlik** [comment]
Please create another issue if you are blocked by this again.
